erpnext/patches/v11_0/add_market_segments.py

```diff
@@ -0,0 +1,17 @@
+
+import frappe
+from frappe import _
+from erpnext.setup.setup_wizard.operations.install_fixtures import make_fixture_records
+
+def execute():
+	frappe.reload_doc('crm', 'doctype', 'market_segment')
+
+	frappe.local.lang = frappe.db.get_default("lang") or 'en'
+
+	records = [
```

Comment:
make a separate function for this and just call it?

Reply:
Of course, I've juste made the split.